test/test_module_init.py

```diff
@@ -231,6 +219,15 @@ def build_constructor_arg_db():
         torch.nn.quantized.FXFloatFunctional: ((), {}),
         torch.nn.quantized.QFunctional: ((), {}),
     }
+    if IS_JETSON:
+        x[torch.nn.qat.Conv2d] = ((3, 3, 3), {'qconfig': torch.ao.quantization.default_qconfig})
+        x[torch.nn.qat.Conv3d] = ((3, 3, 3), {'qconfig': torch.ao.quantization.default_qconfig})
+        x[torch.nn.qat.Linear] = ((5, 2), {'qconfig': torch.ao.quantization.default_qconfig,})
+        x[torch.nn.qat.Embedding] =
+            ((10, 12), {'qconfig': torch.ao.quantization.float_qparams_weight_only_qconfig})
+        x[torch.nn.qat.EmbeddingBag] =
+            ((10, 12), {'qconfig': torch.ao.quantization.float_qparams_weight_only_qconfig})
```

Comment:
Are these tests now only executed on Jetson and skipped otherwise?

Reply:
actually intended to be `not IS_JETSON` which I could have sworn was necessary but upon rechecking it seems these tests are skipped automatically and the changes to this file are not needed.